src/features/common/WebappButton/index.tsx

```diff
@@ -37,6 +37,7 @@ function WebappButton({
         className={styles.webappButtonLink}
       >
         <button
+          onClick={(e) => e.stopPropagation()}
```

Comment:
Move the `onClick` handler to the `Link` element, as we want to stop event propagation beyond the link. 

While the current code seems to work, stopping event propagation at the Link element makes the intention of the code clearer, and we ensure that no click events related to this component (whether they originate from the button or the link) will bubble up. This could prevent subtle bugs in more complex layouts where event bubbling might cause unexpected behavior

Reply:
You've removed the `onClick={(e) => e.stopPropagation()}`, but not added it to the `<Link>`. As a result, the Donate button does not work as intended any more, the user is taken to /prd as the click propagates to the parent container.